pages/inquiry.vue

```diff
@@ -80,6 +80,18 @@ const snackbar = useSnackbar();
 const loading = ref(true);
 const formFields = ref({});
 
+const transformAdapter = (field) => {
+  // If field.options is an array of objects, transform it
+  if (Array.isArray(field.options) && field.options.every(option => typeof option === 'object' && option.hasOwnProperty('key') && option.hasOwnProperty('value'))) {
+    return field.options.reduce((acc, { key, value }) => {
```

Comment:
I think it'd be better to do with simpler fn, like:
```js
const options = {};
field.options.forEach(({ key, value}) => options[key] = value);
return options;
```
Although this suggestion could be based on my preference that `reduce()` is a bit hard for humans to read.
If you accept this please change, otherwise let it be.
How do you think?

Reply:
@yabe-diverta san
I agree. 
Simplified code now.